src/codegen/llvm/jit_driver.cpp

```diff
@@ -80,5 +112,26 @@ std::unique_ptr<llvm::TargetMachine> JITDriver::create_target(
     return std::unique_ptr<llvm::TargetMachine>(tm);
 }
 
+bool JITDriver::set_triple_and_data_layout(const std::string& features) {
+    // Get the deafult target triple for the host.
+    auto target_triple = llvm::sys::getDefaultTargetTriple();
+    std::string error_msg;
+    auto* target = llvm::TargetRegistry::lookupTarget(target_triple, error_msg);
+    if (!target)
+        throw std::runtime_error("Error " + error_msg + "\n");
+
+    // Get the CPU information and set a target machine to create the data layout.
+    std::string cpu(llvm::sys::getHostCPUName());
+
+    std::unique_ptr<llvm::TargetMachine> tm(
+        target->createTargetMachine(target_triple, cpu, features, {}, {}));
+    if (!tm)
+        throw std::runtime_error("Error: could not create the target machine\n");
+
+    // Set data layout and the target triple to the module.
+    module->setDataLayout(tm->createDataLayout());
+    module->setTargetTriple(target_triple);
+    return false;
+}
```

Comment:
return type is not used in `JITDriver::init`. What `false` mean here?

Reply:
Didn't remove it from the previous implementation. Fixed now!